f8a_tagger/tokenizer.py

```diff
@@ -34,19 +36,27 @@ def __init__(self, stopwords_file=None, ngram_size=1, lemmatizer=None, stemmer=N
         self._stemmer = stemmer
         self._lemmatizer = lemmatizer
 
-        with open(stopwords_file or self._STOPWORDS_TXT, 'r') as f:
-            for word in f.read().split('\n'):
-                # Remove comments
-                if word.startswith('#') and word != '#':
-                    continue
+        if isinstance(stopwords_file, str) or stopwords_file is None:
```

Comment:
I'm not sure if I follow here - `stopwords_file` can be also directly io stream (see branch bellow). In this case this branch will not get executed (stopwords_file will not be string and will not be None). Or am I missing something?